zh/routing.md

```diff
@@ -36,7 +36,7 @@ export function createApp () {
     router,
     render: h => h(App)
   })
-  // 注入 router 到根 Vue 实例
```

Comment:
why do you remove the comment?

See the `en` docs:
https://github.com/vuejs/vue-ssr-docs/blame/master/en/routing.md#L44

Reply:
I thought it is a redundancy. Because it is same to line 35. But according to the en version, it should be a translation error.

Reply:
Yes, the 2 comments have different meanings imo. The L39 is about inject and L39 is about return. So agree with @kazupon .